bangazonapi/views/product.py

```diff
@@ -102,7 +103,11 @@ def create(self, request):
 
             new_product.image_path = data
 
-        new_product.save()
+        try:
+            new_product.clean_fields()
+            new_product.save()
+        except ValidationError as ex:
+            return Response({'message': ex.args[0]}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
Everything is found at this point, it's just not valid. I'd argue a 404 is not what we want here. See what I mean?

Consider something like a `HTTP_400_BAD_REQUEST`.

Reply:
So what you need .. when  I test give me this result :: {
    "message": {
        "price": [
            [
                "Ensure this value is less than or equal to 17500.0."
            ]
        ]
    }
}

so that should be good.

Reply:
The response message is correct. The status code is not.